project/Build.scala

```diff
@@ -1354,35 +1356,20 @@ object Build {
               otherProducts.flatMap(base => Path.selectSubpaths(base, filter))
 
             libraryMappings ++ otherMappings
-          }
-      ))
-  ).withScalaJSCompiler
-
-  lazy val minilib: MultiScalaProject = MultiScalaProject(
-      id = "minilib", base = file("minilib")
-  ).enablePlugins(
-      MyScalaJSPlugin
-  ).settings(
-      commonSettings,
-      fatalWarningsSettings,
-      name := "scalajs-minilib",
-
-      noClassFilesSettings,
-      scalaJSExternalCompileSettings,
-  ).zippedSettings(library)(library =>
-      inConfig(Compile)(Seq(
-          mappings in packageBin := {
-            val superMappings = (mappings in packageBin).value
-            val libraryMappings = (mappings in (library, packageBin)).value
+          },
 
-            val whitelisted = libraryMappings.filter { mapping =>
+          packageMinilib := {
```

Comment:
Have you considered using `Defaults.packageTaskSettings`? (I'm not sure it'll turn out simpler, just wondering if you have looked at it).

Reply:
I have looked at it, and it gets awfully more complicated ;)

Reply:
That's was my hunch as well :)